client/spaceList.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Spaces | Makers BnB</title>
+  </head>
+  <body>
+
+    <h1> List of spaces <span id="space-list"></span>
+      <ul>
+        <li id="spaces"></li>
```

Comment:
What happens if you remove this list element? Not sure what it's doing - can we get rid?